include/bbp/sonata/config.h

```diff
@@ -203,10 +203,28 @@ class SONATA_API SimulationConfig
      * List of report parameters collected during the simulation
      */
     struct Report {
+        enum class Sections { invalid = -1, soma, axon, dend, apic, all };
+        enum class Type { invalid = -1, compartment, summation, synapse };
+        enum class Scaling { invalid = -1, none, area };
+        enum class Compartments { invalid = -1, center, all };
+
         /// Node sets on which to report
         std::string cells;
+        /// Sections on which to report.
+        /// Possible values: "soma"(default), "axon", "dend", "apic", "all"
```

Comment:
Only need to say what the defaults are, not the possible values; the enum describes that